components/org.wso2.carbon.identity.organization.management.service/src/main/java/org/wso2/carbon/identity/organization/management/service/constant/OrganizationManagementConstants.java

```diff
@@ -756,7 +756,10 @@ public enum ErrorMessages {
                 "Server encountered an error while checking organization user associations for the user."),
         ERROR_CODE_ERROR_CHECKING_ORGANIZATION_EXIST_BY_HANDLE("65145",
                 "Error while checking if the organization exists.",
-                "Server encountered an error while checking if the organization with handle: %s exists.");
+                "Server encountered an error while checking if the organization with handle: %s exists."),
+        ERROR_CODE_ERROR_RETRIEVING_ORGANIZATION_NAMES_IN_BATCH("65146",
```

Comment:
Let's make this specific without saying IN_BATCH. 
Like `ERROR_CODE_ERROR_RETRIEVING_ORGANIZATION_NAMES_BY_ORGANIZATION_IDS`

Reply:
Also update the error message as well. Error description seems to be fine

Reply:
Also this should be updated accordingly if the comment https://github.com/wso2/identity-organization-management-core/pull/174/files#r2059483032 applies